intellij-plugin-verifier/verifier-test/src/test/java/com/jetbrains/pluginverifier/tests/bytecode/ByteCodeDescriptions.kt

```diff
@@ -0,0 +1,127 @@
+package com.jetbrains.pluginverifier.tests.bytecode
+
+import org.objectweb.asm.Opcodes.*
+import org.objectweb.asm.tree.InsnNode
+import org.objectweb.asm.tree.TypeInsnNode
+
+@Suppress("unused")
+fun TypeInsnNode.toDebugString(): String = when (opcode) {
```

Comment:
Why `toDebugString()` are needed? Are they used implicitly? Maybe it would be good to explain it above "unused" suppressions.

Reply:
Added a KDoc to indicate that this is used in IDE debugging sessions.